### ISSUE TITLE
Blind users with screenreader read everything in German

### ISSUE THREAD

**Issue by tonyfast** [jupyter/notebook#6924]
### Language

- **Expectation**: Language tags are specified on `<html>` element based on the language the content is being displayed as. If content is in English, needs to be `<html lang="en">`, etc. (WCAG Criteria 3.1.1 (A) and 3.1.2 (AA))
- **Observed**:
  - On all pages with my language set to English, `lang="default"` which reads my content in GERMAN (probably looking at the first 2 letters "de"). Need to have JS that powers this instead change to the proper lang codes.
  - If importing over the Settings Editor to Notebookv7 ever, recommend adding dropdown menu for languages so user only selects from list of languages that are currently supported/installed so bilingual blind users can easily toggle without fear of messing up.

**Comment by tonyfast** [comment]
i spent some time looking for where the `lang` attribute is set on the application. i am sure this has some overlapping complications with the translation module.

this one should be easy to solve for somewho knows where the `lang` attribute is set.

**Comment by andrii-i** [comment]
Thank you for creating this issue @tonyfast

**Comment by andrii-i** [comment]
Problem seems to be in the lab (https://github.com/jupyterlab/jupyterlab/issues/14704), opened PR to fix it there https://github.com/jupyterlab/jupyterlab/pull/14705

**Comment by andrii-i** [comment]
https://github.com/jupyterlab/jupyterlab/pull/14705 was merged. Next steps would be releasing/waiting for JypyterLab version with 14705 included and updating Notebook to it.

**Comment by tonyfast** [comment]
closing this as completed then.
